data-node/src/main/java/org/graylog/datanode/Configuration.java

```diff
@@ -207,7 +207,7 @@ public void validatePasswordSecret() throws ValidationException {
     private String restApiPassword;
 
     @Parameter(value = "node_id_file", validators = NodeIdFileValidator.class)
-    private String nodeIdFile = "/etc/graylog/datanode/node-id";
+    private String nodeIdFile = "./bin/config/node-id";
```

Comment:
Let's change the node ID file location to the data dir.
```suggestion
    private String nodeIdFile = "data/node-id";
```